receiver/hostmetricsreceiver/internal/scraper/processscraper/get_process_handles_windows.go

```diff
@@ -0,0 +1,84 @@
+// Copyright The OpenTelemetry Authors
+// SPDX-License-Identifier: Apache-2.0
+
+//go:build windows
+
+package processscraper // import "github.com/open-telemetry/opentelemetry-collector-contrib/receiver/hostmetricsreceiver/internal/scraper/processscraper"
+
+import (
+	"context"
+	"fmt"
+	"unsafe"
+
+	"github.com/shirou/gopsutil/v4/process"
+	"go.opentelemetry.io/collector/featuregate"
+	"golang.org/x/sys/windows"
+)
+
+var useNewGetProcessHandles = featuregate.GlobalRegistry().MustRegister(
+	"hostmetrics.process.OnWindowsUseNewGetProcesses",
```

Comment:
I'm not sure if we have a naming convention for these, but I think usually we don't capitalize the first letter, so it would be `onWindowsUseNewGetProcesses`. I'm not sure if that's a hard and fast rule written somewhere or not.

Reply:
4ae28a72d33acffdc48bc6442b29d4d52e3cafbc